include/audit_events.hpp

```diff
@@ -37,20 +41,49 @@ inline void auditEvent(const char* opPath, const std::string& userName,
         return;
     }
 
-    strncpy(cnfgBuff, opPath, std::strlen(opPath) + 1);
+    opPathLen = std::strlen(opPath) + 1;
+    if (opPathLen > bufLeft)
```

Comment:
Maybe a comment if you want, explaining why we need to do this

Reply:
Good point. I pushed another commit with a couple of comments.